src/key_packages/mod.rs

```diff
@@ -237,6 +237,18 @@ pub struct KeyPackageBundle {
     pub(crate) private_key: HPKEPrivateKey,
 }
 
+impl Clone for KeyPackageBundle {
+    fn clone(&self) -> Self {
+        KeyPackageBundle {
+            key_package: self.key_package.clone(),
+            // This requires the hpke hazmat feature, which is not something
+            // that should ordinarily be used, as we generally try to avoid
+            // cloning private key material.
+            private_key: HPKEPrivateKey::new(self.private_key.as_slice().to_vec()),
+        }
+    }
+}
+
```

Comment:
Do we need that? With the `PrivateTree` refactoring KeyPackageBundles can be passed as a reference.

Reply:
I'll look into it.

Reply:
The clone is indeed not needed anymore :slightly_smiling_face: